fontir/src/ir.rs

```diff
@@ -189,7 +189,28 @@ impl GlyphPathBuilder {
     }
 
     pub fn qcurve_to(&mut self, p: impl Into<Point>) -> Result<(), PathConversionError> {
-        todo!("Support qcurve to {}", p.into());
+        // https://github.com/googlefonts/fontmake-rs/issues/110
+        // Guard clauses: degenerate cases
+        if self.is_empty() {
+            return self.move_to(p);
+        }
+        if self.offcurve.is_empty() {
+            return self.line_to(p);
+        }
+
+        // Insert an implied oncurve between every pair of offcurve points
+        for i in 0..(self.offcurve.len() - 1) {
```

Comment:
would `self.offcurve.windows(2)` be useful here?

Reply:
It would! So updated.